bot/seasons.py

```diff
@@ -30,7 +30,7 @@ class SeasonBase:
 
 
 class Christmas(SeasonBase):
-    """Branding for december."""
+    """Branding for December."""
 
     season_name = "Festive season"
     bot_name = "Merrybot"
```

Comment:
It bothers me that all names had bot capitalised besides this one.

```suggestion
    bot_name = "MerryBot"
```

Reply:
Adjusted in d575995.